docs/kb/kdb-tick.md

```diff
@@ -1,55 +1,128 @@
+Don’t forget though that a second RDB will increase the memory usage, as it's an in-memory database and can’t be sharing any data with the primary RDB.
 
-The TP logs the updates published to subscribers to a file. In the event of a serious crash, this file can be rescued using the utility functions in 
-:fontawesome-brands-github: 
-[simongarland/tickrecover/rescuelog.q](https://github.com/simongarland/tickrecover/blob/master/rescuelog.q)
+#### Example
+
+The example script [`chainedr.q`](https://github.com/KxSystems/kdb/blob/master/tick/chainedr.q) can be run as follows:
+```bash
+q chainedr.q [host]:port[:usr:pwd] [-p 5111]
+```
+
+If a tickerplant is running on the same host (port 5010), the following starts a chained RDB on port 5111
+```q
+$ q chainedr.q :5010 -p 5111 
+```
+
+:fontawesome-brands-github: [KxSystems/kdb/tick/chainedr.q](https://github.com/KxSystems/kdb/blob/master/tick/chainedr.q)
+
+### Write-only RDB
+
+The default behavior of the RDB is to collect data to an in-memory database during the day and then to save it to disk as an historical partition at day end. 
+This makes sense if it’s actually queried during the day, but if the only reason for having an RDB is to be able to save the historical partition 
+the amount of memory required to keep the in-memory database can be excessive.
+
+It would make sense to write the data to disk during the day so that it’s ready for day-end processing, but with only a small memory footprint to build bulk updates.
+
+:fontawesome-brands-github:
+[simongarland/tick/w.q](https://github.com/simongarland/tick/blob/master/w.q)
+is a potential replacement for the default RDB
+:fontawesome-brands-github:
+[KxSystems/kdb-tick/tick/r.q](https://github.com/KxSystems/kdb-tick/blob/master/tick/r.q).
+
+`w.q` connects to the tickerplant just like `r.q`, but it buffers up requests and, every `MAXROWS` records, writes the data to disk. 
+At day end, remaining data is flushed to disk, the database is sorted (on disk) and then moved to the appropriate date partition within the historical database.
+
+Note that it makes no sense to query the task running `w.q` as it will have a small (and variable-sized) selection of records. 
+Although it wouldn’t be difficult to modify it to keep say the last 5 minutes of data, 
+that sort of custom collection is probably better housed in a task running a [`c.q`](#cq)-like aggregation.
+
+Syntax:
+```bash
+q w.q [tickerplanthost]:port[:usr:pwd] [hdbhost]:port[:usr:pwd] [-koe|keeponexit]
+```
+
+e.g.
+```bash
+$ q w.q :5010 :5012
+```
+
+The `-koe` or `-keeponexit` parameter governs the behavior when a `w.q` task is exited at user request i.e. `.z.exit` is called. 
+By default the data saved so far is deleted, as if the task were restarted it would be difficult to ensure it restarted from exactly the right place.
+It’s easier to replay the log and (re)write the data. If the flag is provided (or the `KEEPONEXIT` global set to `1b`) the data will not be removed.
```

Comment:
```suggestion
The `-koe` or `-keeponexit` parameter governs the behavior when a `w.q` task is exited at user request, when `.z.exit` is called. 
By default, the data saved so far is deleted. If the task were restarted it would be difficult to ensure it restarted from exactly the right place.
It’s easier to replay the log and (re)write the data. If the flag is provided (or the `KEEPONEXIT` global set to `1b`) the data is not removed.
```